e_os.h

```diff
@@ -17,56 +17,6 @@
 # include <openssl/crypto.h>
 # include "internal/nelem.h"
 
-/*
- * <openssl/e_os2.h> contains what we can justify to make visible to the
- * outside; this file e_os.h is not part of the exported interface.
- */
```

Comment:
Do not remove this comment

Reply:
Yeah, your're right. I did not look close enough when moving the block.

Reply:
(resolved: comment was restored some time ago)